common/src/main/resources/plugin.yml

```diff
@@ -1,11 +1,10 @@
 name: TerraformGenerator
 author: Hex_27
 version: 3.1.2
-depend: [ Drycell ]
 api-version: 1.14
 description: World Generator
 main: org.terraform.main.TerraformGeneratorPlugin
-load: startup
+load: STARTUP
```

Comment:
Does it really matter that this is uppercase?

Reply:
While yes it *works* if its lowercase, the schema specifies that it is uppercase, so I changed it. Why must we split hairs over this?

Reply:
Just saw it as a weird change to make since it wouldn't change things. Not much of an issue for me. Just odd, that's all.